frontend/public/style/_forms.scss

```diff
@@ -77,6 +77,15 @@
   padding-right: 25px;
 }
 
+.pf-c-form__actions--right {
+  justify-content: flex-end;
+}
+
 .pf-c-form__group--no-top-margin {
   margin-top: 0 !important;
 }
+
+// specificity to override pf4
+.pf-c-form__group.pf-m-action.pf-c-form__actions--no-top-margin {
```

Comment:
I'm unclear why this is needed.  Can you not simply use `pf-c-form__group--no-top-margin` instead?

Reply:
pf4 sets `margin-top` to `.pf-c-form__group.pf-m-action` 
I'll need to set `.pf-c-form__actions--no-top-margin` `!important` to override.